pytket/binders/circuit/Circuit/main.cpp

```diff
@@ -413,6 +413,9 @@ void def_circuit(py::class_<Circuit, std::shared_ptr<Circuit>> &pyCircuit) {
       .def_property_readonly(
           "n_gates", &Circuit::n_gates,
           ":return: the number of gates in the Circuit")
+      .def_property_readonly(
+          "wasm_uid", &Circuit::get_wasm_file_uid,
+          ":return: the wasm uid of the first wasmop found in the circuit")
```

Comment:
Can a circuit contain more than one WASM UID? If so it would be preferable to return the full set here. If not, we can be explicit about this in the doc.